src/moderngl.cpp

```diff
@@ -7530,10 +7530,10 @@ PyObject * MGLTextureCube_get_swizzle(MGLTextureCube * self, void * closure) {
 int MGLTextureCube_set_swizzle(MGLTextureCube * self, PyObject * value, void * closure) {
     const char * swizzle = PyUnicode_AsUTF8(value);
 
-	if (self->depth) {
-		MGLError_Set("cannot get swizzle of depth textures");
-		return 0;
-	}
+    if (self->depth) {
+        MGLError_Set("cannot get swizzle of depth textures");
+        return 0;
```

Comment:
errors in a setter must return -1 (https://docs.python.org/3/c-api/structures.html#c.PyGetSetDef)